jpasskit/src/main/java/de/brendamour/jpasskit/apns/PKSendPushNotificationUtil.java

```diff
@@ -33,45 +34,55 @@
 import com.turo.pushy.apns.util.concurrent.PushNotificationFuture;
 import com.turo.pushy.apns.util.concurrent.PushNotificationResponseListener;
 
+import org.apache.commons.lang3.tuple.Pair;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static de.brendamour.jpasskit.util.Assert.state;
```

Comment:
Not a big fan of static imports. Makes testing much more cumbersome.

Reply:
When I see "testing", I think about unit tests. Is it what you meant? Cause I don't know of such limitations - I'd appreciate if you could explain them to me.

If you tell about debugging and overall code readability - this is quite a subjective opinion, but I'm fine with following your code style. As I understand, this code style accepts static imports in test code, but does not recommend it for the main one.

Please correct me if I'm wrong.

Reply:
static imports can't be easily mocked in unit tests making it hard to test things in isolation (=> unit tests)

Reply:
Mmm... I understand your idea - I did not have such issues myself, but it makes sense.

So... we allow static imports only for test code, right? No need to update test files, I mean.

Reply:
yeah

Reply:
Done!